components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2255,8 +2255,11 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
         try {
             const billingMode = await this.billingModes.getBillingModeForUser(user, new Date());
             if (billingMode.mode === "usage-based") {
-                const limit = await this.billingService.checkUsageLimitReached(user);
                 let teamOrUser;
```

Comment:
This can be made `const` and moved into the switch case on line 2270 where it is used.